sign.js

```diff
@@ -150,14 +171,15 @@ function processAddress(path, last) {
     throw new Error('Public key mismatch: ' + pub.toString());
   }
 
-  var script = bitcore.Script.buildMultisigOut(publicKeys, walletMetadata.credentials.m);
+  const m = legacyCoPay ? walletMetadata.m : walletMetadata.credentials.m;
```

Comment:
if you do 
```
const m = walletMetadata.m || walletMetadata.credentials.m;
```
you wouldnt need to pass legacy copay in there as argument

Reply:
good idea, changed